roboflow/util/clip_compare_utils.py

```diff
@@ -0,0 +1,48 @@
+import base64
+import glob
+import io
+import json
+
+import requests
+from PIL import Image
+
+from roboflow.config import CLIP_FEATURIZE_URL
+
+
+# rf.predict requires images formatted and base64 encoded
+def base64_encode(image_path):
+    """
+    @params:
+        iamge_path: (str) = name reference to a given image for encoding
+
+    returns: a base64 encoded string formatted for travel to an HTTP endpoint
+    """
+
+    image = Image.open(image_path)
+    buffered = io.BytesIO()
+    image_rgb = image.convert("RGB")
+    image_rgb.save(buffered, quality=90, format="JPEG")
```

Comment:
why JPEG with a lower quality?

Reply:
I ripped this out of some sample code from deeper in the code base and didn't consider changing the `format`. What should we be using as a standard? I think `.tiff` files technically have the best quality but not sure if we should be using something more common such as `.png`.

Reply:
If we don't know why this is done, let's just store it as png with untouched quality